packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -656,7 +657,7 @@ export class RunningSummarizer implements IDisposable {
 					);
 					const result = await resultSummarize.receivedSummaryAckOrNack;
 
-					if (result.success) {
+					if (result.success || result.error === restartErrorMessage) {
```

Comment:
As per my comment https://github.com/microsoft/FluidFramework/pull/15036#issuecomment-1507303691, we should not do this. The second summary attempt will always fail so don't even do it.